spk/ntopng/src/start-stop-status.sh

```diff
@@ -0,0 +1,186 @@
+#! /bin/bash
+
+# start/stop script derived from ntopng/packages/etc/init.d/ntopng
+
+SERVICE_CFG_FILE="${SYNOPKG_PKGDEST}/etc/ntopng.conf"
+SERVICE_COMMAND="${SYNOPKG_PKGDEST}/bin/ntopng ${SERVICE_CFG_FILE}"
```

Comment:
According to comment in `service-setup.sh`, you have planned to source variables from there. By the way, I think you may discard `service-setup.sh` completely without any trouble.

Reply:
Sorry, forgot to push this (914b928)